File: generated_code_examples/r/classification/random_forest.r
score <- function(input) {
    return(((subroutine0(input)) * (0.5)) + ((subroutine1(input)) * (0.5)))
}
subroutine0 <- function(input) {
    if ((input[4]) <= (0.8)) {
        var0 <- c(1.0, 0.0, 0.0)
    } else {
        if ((input[3]) <= (4.8500004)) {
            var0 <- c(0.0, 0.9622641509433962, 0.03773584905660377)
        } else {
            if ((input[4]) <= (1.75)) {
                if ((input[4]) <= (1.6500001)) {
                    var0 <- c(0.0, 0.25, 0.75)
                } else {
                    var0 <- c(0.0, 1.0, 0.0)
                }
            } else {
                var0 <- c(0.0, 0.0, 1.0)
            }
        }
    }
    return(var0)
}
subroutine1 <- function(input) {
    if ((input[4]) <= (0.8)) {
        var0 <- c(1.0, 0.0, 0.0)
    } else {
        if ((input[1]) <= (6.1499996)) {
            if ((input[3]) <= (4.8500004)) {
                var0 <- c(0.0, 0.9090909090909091, 0.09090909090909091)
            } else {
                var0 <- c(0.0, 0.0, 1.0)
            }
        } else {
            if ((input[4]) <= (1.75)) {
                var0 <- c(0.0, 0.8666666666666667, 0.13333333333333333)
            } else {
                var0 <- c(0.0, 0.0, 1.0)
            }
        }
    }
    return(var0)
}

File: generated_code_examples/f_sharp/regression/random_forest.fs
let score (input : double list) =
    let func0 =
        if ((input.[12]) <= (8.91)) then
            if ((input.[5]) <= (6.902)) then
                if ((input.[7]) <= (1.48495)) then
                    50.0
                else
                    25.320000000000004
            else
                38.34810126582279
        else
            if ((input.[0]) <= (5.84803)) then
                19.99185520361991
            else
                12.102469135802467
    let func1 =
        if ((input.[12]) <= (9.725)) then
            if ((input.[5]) <= (7.4525)) then
                if ((input.[5]) <= (6.7539997)) then
                    24.801739130434775
                else
                    32.47230769230769
            else
                47.075
        else
            if ((input.[12]) <= (15.0)) then
                20.44094488188976
            else
                14.823214285714291
    ((func0) + (func1)) * (0.5)
